low-level/framework/utils/service_logging.py

```diff
@@ -19,130 +19,23 @@
  ****************************************************************************
 """
 
-import logging.handlers
-import time
+import sys
 import os
+from cortx.utils.log import Log
 
+def init_logging(dcs_service_name, file_path, log_level="INFO"):
+    """Initialize logging for SSPL component."""
+    try:
+        Log.init(service_name=dcs_service_name, log_path=file_path, level=log_level)
+    except Exception as err:
+        print("[ Error ] when initializing logging :")
```

Comment:
Push these error statements to system syslog in case of exceptions. Also optimize the statement
>"[ Error ] CORTX Logger Init failed with error {err}"

Reply:
Done